backend/pkg/api/data_access/user.go

```diff
@@ -361,6 +367,61 @@ func (d *DataAccessService) GetUserDashboards(ctx context.Context, userId uint64
 		return nil
 	})
 
+	// TODO could merge with above query
+	validatorDashboardEBs := make(map[uint64]uint64, 0)
+	wg.Go(func() error {
+		validatorsDs := goqu.Dialect("postgres").
+			From(goqu.T("users_val_dashboards").As("uvd")).
+			LeftJoin(
+				goqu.T("users_val_dashboards_validators").As("uvdv"),
+				goqu.On(goqu.I("uvd.id").Eq(goqu.I("uvdv.dashboard_id"))),
+			).
+			Select(
+				goqu.I("uvd.id").As("dashboard_id"),
+				goqu.I("uvdv.validator_index").As("validator_index"),
+			).
+			Where(goqu.I("uvd.user_id").Eq(userId))
+
+		validatorsQuery, args, err := validatorsDs.Prepared(true).ToSQL()
+		if err != nil {
+			return err
+		}
+
+		// could also scan into array
+		dashboardValidators := []struct {
+			Id             uint64 `db:"dashboard_id"`
+			ValidatorIndex uint64 `db:"validator_index"`
+		}{}
+		err = d.alloyReader.SelectContext(ctx, &dashboardValidators, validatorsQuery, args...)
+		if err != nil {
+			return err
+		}
+		dashboardValidatorsMap := make(map[uint64][]t.VDBValidator, 0)
+		validators := make([]t.VDBValidator, 0, len(dashboardValidators))
+		for _, row := range dashboardValidators {
+			dashboardValidatorsMap[row.Id] = append(dashboardValidatorsMap[row.Id], row.ValidatorIndex)
+			validators = append(validators, row.ValidatorIndex)
+		}
+		validatorEbs, err := d.GetValidatorsEffectiveBalances(ctx, validators, false)
+		if err != nil {
+			return err
+		}
+
+		for dashboard_id, validators := range dashboardValidatorsMap {
+			var dashboardTotalEb uint64
+			for _, validator := range validators {
+				if eb, ok := validatorEbs[validator]; !ok {
+					return fmt.Errorf("effective balance for validator %d not found", validator)
+				} else {
+					dashboardTotalEb += eb
+				}
```

Comment:
Nitpick: Unnecessary else
```suggestion
				dashboardTotalEb += eb
```

Reply:
also applied in https://github.com/gobitfly/beaconchain/pull/1367